smart-doc/_DC-article

```diff
@@ -0,0 +1,18 @@
+# This file originates from the project https://github.com/openSUSE/doc-kit
+# This file can be edited downstream.
+
+MAIN="assembly.asm.xml"
+# Point to the ID of the <structure> of your assembly
+ROOTID="article-example"
+SRC_DIR="articles"
+IMG_SRC_DIR="images"
+
+PROFCONDITION="suse-product"
+#PROFCONDITION="suse-product;beta"
+#PROFCONDITION="community-project"
+
+# Just a test:
```

Comment:
Probably not so anymore. :wink: I'd remove this comment:

```suggestion
```

Reply:
Applied in https://github.com/openSUSE/doc-kit/pull/78.